### ISSUE TITLE
Cannot find device. Empty Array.

### ISSUE THREAD

**Issue by ermirshabani1** [farellsujanto/flutter_serial_communication#12]
It doesnt find my device when i run getAvailableDevices. It returns empty array.
But in another project that runs in java, when i try to get my device it actually finds the available port everytime.

**Comment by qamarkhan1** [comment]
Yes I have in device_filter.xml and camera is connected via OTG, but it shows no device found. Same camera is found and comes to available devices when I use any of these 2 packages from pub dev 1) flutter_uvc_camera and 2) uvccamera. But I just want to discover my device so I want to utilize this package, let me me if you can assist

**Comment by shreeneet** [comment]
@qamarkhan1 did you figure this out?
